types/puppeteer/index.d.ts

```diff
@@ -414,7 +416,6 @@ export interface DeleteCookie {
   url?: string;
   domain?: string;
   path?: string;
-  secure?: boolean;
```

Comment:
If you apply fixes, please also note these in your PR message to avoid confusion as those were not in the changelogs.